pkg/cmdutils/config.go

```diff
@@ -125,6 +126,29 @@ func (c *ClusterConfig) addBKFlags(flags *pflag.FlagSet) {
 	)
 }
 
+func (c *ClusterConfig) addOAuth2Flags(flags *pflag.FlagSet) {
```

Comment:
Please ensure that these settings are overridable using the context feature.  Each Pulsar instance has a different audience, and may have a different issuer endpoint, client-id, or keyfile.  Note that, in SN Cloud, only the audience would be different for each Pulsar instance (in most cases).

Reply:
Sure. Added it to the context. https://github.com/streamnative/pulsarctl/pull/233/commits/07ac2d936d09042c2abbb2049a576203b3827285#diff-2f50108608ddee9263c2bc96f7373770R55